pkg/utils.go

```diff
@@ -240,6 +240,11 @@ func GetTillerStorage(tillerNamespace string) string {
 			storage = "secrets"
 		}
 	}
+	for _, c := range pods.Items[0].Spec.Containers[0].Args {
```

Comment:
would it make sense to get the container first?
something along the lines of:
```
container := pods.Items[0].Spec.Containers[0]
for _, c := container.Command { ... }
for _, a := container.Args { ... }
```

Reply:
Sure that makes sense, thanks.